docs/featurestore/featurestore.rst

```diff
@@ -843,10 +849,57 @@ Hopsworks relies on certificates being available in the Databricks cluster in or
 In the Databricks notebooks connected to the prepared cluster use the following code to connect to the feature store::
 
     import hops.featurestore as fs
-    fs.connect('my_instance.us-east-2.compute.amazonaws.com', 'my_project', cert_folder='/dbfs/mnt/my_mount_name')
+    fs.connect('my_instance', 'my_project', region_name='my_aws_region', cert_folder='/dbfs/certs/')
++------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Replace *my_instance* by the address of your instance, *my_project* by the name of your project and *my_aws_region* by the AWS region in which you stored the API Key. |
++------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
 
 If you have trouble connecting, then ensure that the Security Group of your Hopsworks instance on AWS is configured to allow incoming traffic from your SageMaker instance. See `VPC Security Groups <https://docs.aws.amazon.com/vpc/latest/userguide/VPC_SecurityGroups.html>`_. for more information. If the Hopsworks instance is not accessible from the internet then you will need to configure `VPC Peering <https://docs.databricks.com/administration-guide/cloud-configurations/aws/vpc-peering.html>`_.
 
+**Installing hops-util-py**
+
+.. warning:: 
+ - This is an advanced setup, if you are not sure that you need it or if you are not sure of what you are doing, please use hopsworks-cloud-sdk. 
+ - Hopsworks-cloud-sdk needs to be able to connect directly to the ip on which the hops name node and hive metastore are listening. This, for example, means that if you deployed Hopsworks on AWS you will need to set up `VPC Peering <https://docs.databricks.com/administration-guide/cloud-configurations/aws/vpc-peering.html>`_ between your Databricks cluster and the Hopsworks one.
+
+In the Databricks UI, go to *Clusters* and select your cluster. Select *Libraries*. Make sure that hopsworks-cloud-sdk is not installed (uninstall it if it is). Then *Install New*. As *Library Source* choose *PyPI* and fill in *hops-util-py* into the *Package* field.
```

Comment:
`(uninstall it if it is)` sounds weird. Maybe `make sure to uninstall it if that's the case`

Reply:
done